CHANGELOG.md

```diff
@@ -47,6 +47,7 @@
 #### Changes
```

Comment:
The following needs to be added to the changelog

#### Additions
* Added support for the plugin "HuskTowns"

#### Fixes
* Fixed "LogBlock" integration
* Fixed "Lands" integration

Reply:
Alright, didn't think you'd want them added here since it was in dough but will do